### PR TITLE
Fix/5389 The headline "Ihr Einverständnis" on the Data Donation Screen behaves strangely

### PR DESCRIPTION
**Pull request by tcfos**
## Description
The headline "Ihr Einverständnis" is displayed after a short delay. Affected screens:

- Onboarding – Data Donation
- Setting > Data Donation

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5389

## Screen Recording 
Onboarding > Data Donation: headline "Ihr Einverständnis" is displayed without any delay.

https://user-images.githubusercontent.com/7896005/111324675-43ad8500-866b-11eb-9fff-ba419375a0d6.mov

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewController.swift [hunk 1]
```diff
@@ -62,7 +62,7 @@ class DataDonationViewController: DynamicTableViewController, DeltaOnboardingVie
 			.sink { [weak self] _ in
 				guard let self = self else { return }
 				self.dynamicTableViewModel = self.viewModel.dynamicTableViewModel
-				DispatchQueue.main.asyncAfter(wallDeadline: .now() + 0.35) {
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewController.swift:65]: I really wonder why this 0.35 was there in the first place. Seems quite specific.

Comment (tcfos) [reply src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewController.swift:65]: Checked already with Kai. In the past he discovered obscure timing issues but today testing didn't show any issues.

Comment (KaiTeuber) [reply src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewController.swift:65]: maybe from some testing while development, I can't remember why

**Review by KaiTeuber — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewController.swift (hunk 1 above)
Comment (marcussc) [reply src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewController.swift:65]: Okay do we still need the Dispatch then?

Comment (tcfos) [reply src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewController.swift:65]: removed the obsolete dispatch

Comment (tcfos) [reply src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewController.swift:65]: Good point. Removed the obsolete dispatch.

**Review by marcussc — APPROVED** [review]

**Pull request merged**